Misc/NEWS.d/next/Security/2021-02-14-15-59-16.bpo-42967.YApqDS.rst

```diff
@@ -0,0 +1 @@
+Fix web cache poisoning vulnerability by defaulting the query args separator to `&`, and allowing the user to choose a custom separator.
```

Comment:
```suggestion
Fix web cache poisoning vulnerability by defaulting the query args separator to ``&``, and allowing the user to choose a custom separator.
```

Single backticks mean a reST role (default one if configured, or error), double backticks are for code bits.